reflect.go

```diff
@@ -1184,3 +1184,12 @@ func (v Value) Uint() uint64 {
 func (v Value) UnsafeAddr() uintptr {
 	return value_UnsafeAddr(v)
 }
+
+// SetExport sets v's flag to fliped of flagRO.
+func (v Value) SetExport() Value {
```

Comment:
I propose `AllowCall` as name of this API . How about it ?

Reply:
May I validate first with `v.mustBe(Func)` ?
( But I know `mustBe` is a private API and it doesn't linked this package ... )